### DISCUSSION TITLE
How do I calculate hardware requirements?

### DISCUSSION THREAD

**Discussion by kadirnar** [huggingface/autotrain-advanced#599] [General]
Hi, I want to fine-tune the llama3-70B model using the autotrain library. How do I calculate the hardware requirement for this? Does the Autotrain library use qlora optimization? For other libraries, 48 or 80GB of vram is sufficient.

**Comment by abhishekkrthakur** [comment]
autotrain uses qlora by default. you can find all the parameters here: https://huggingface.co/docs/autotrain/llm_finetuning
for 70b using qlora, you will need ~2 GPUs. 2xA10g should be good. ive tested on A100s and H100s though.
